docs/source/augly.text.augmenters.rst

```diff
@@ -0,0 +1,109 @@
+augly.text.augmenters package
```

Comment:
borderline about `augmenters` too -- they're not toplevel classes, they're for development purposes

Reply:
since we deleted the other augmenters .rst files, let's delete this one too for consistency